utils/__tests__/Navigation.test.js

```diff
@@ -0,0 +1,137 @@
+        children: [
+          {
+            title: 'Reference',
+            to: '/reference',
+            icon: 'receipt',
+            children: [
+              {
+                title: 'Merchant Integrations',
+                to: '/guides/line-items',
+                children: [
+                  {
+                    title: 'Line Items',
+                    to: '/guides/line-items',
+                    children: [],
+                  },
+                  {
+                    title: 'overridden title',
+                    to: '/guides/point-of-sale',
+                    children: [],
+                  },
+                ]
+              }
+            ]
+          },
+          {
+            title: 'Connections',
+            to: '/connections',
+            icon: 'connections',
+            children: [
+              {
+                title: 'Farmlands',
+                to: '/connections/farmlands',
+                children: [
+                  {
+                    title: 'POS Integration Guide',
+                    to: '/guides/farmlands-pos-integration',
+                    children: [],
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            title: 'API',
+            to: 'https://www.baseurl/api',
+            icon: 'settings',
+            children: [],
+          }
+        ]});
+    });
+  });
+
+  describe('getCurrentNavPath', () => {
+    it('should return nav path when the current route has no trailing slash', () => {
+      Navigation.create({ baseUrl: 'https://www.baseurl', content: [] });
+      const result = Navigation.getCurrentNavPath('/connections/farmlands');
+      expect(result).toEqual('/connections/farmlands');
+    });
+
+    it('should strip trailing slash from nav path when the current route has a trailing slash', () => {
+      Navigation.create({ baseUrl: 'https://www.baseurl', content: [] });
+      const result = Navigation.getCurrentNavPath('/connections/farmlands/');
+      expect(result).toEqual('/connections/farmlands');
+    });
+
+    it('should map current route to nav path when the current route must be mapped', () => {
+      Navigation.create({
+        baseUrl: 'https://www.baseurl',
+        content: [
+          {
+            _path: '/guides/farmlands-pos-integration',
+            title: 'Farmlands POS Integration',
+            nav: {
+              path: '/connections/farmlands',
+              order: 1
+            },
+          },
+        ]
+      });
+      const result = Navigation.getCurrentNavPath('/guides/farmlands-pos-integration');
+      expect(result).toEqual('/connections/farmlands/farmlands-pos-integration');
+    });
+
+    it('should not map current route to nav path when the current route does not have to be mapped', () => {
+      Navigation.create({
+        baseUrl: 'https://www.baseurl',
+        content: [
+          {
+            _path: '/connections/farmlands/farmlands-pos-integration',
```

Comment:
It was hard to understand what this test was testing - if we're just testing the default return value for `getCurrentNavPath`, perhaps this could be simpler/clearer?

Reply:
I have removed that test, it is not testing anything that the test wasn't testing.